common/changes/@microsoft/rush/ianc-configureable-git-path_2021-01-08-00-54.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix an issue where git operations were performed even if --target-branch wasn't provided to rush publish or rush version.",
```

Comment:
Can you update the PR description with a more detailed explanation of (1) what was broken and (2) what you changed to fix it?

The comment here is rather vague.

Reply:
I ended up getting rid of this.